source/features/conversation-links-on-repo-lists.tsx

```diff
@@ -46,15 +47,8 @@ void features.add({
 }, {
 	include: [
 		pageDetect.isUserProfileRepoTab,
-		pageDetect.isGlobalSearchResults
-	],
-	init
-}, {
-	include: [
+		pageDetect.isGlobalSearchResults,
 		pageDetect.isUserProfileRepoTab
```

Comment:
Duplicate page detect.

Reply:
I feel a Lint PR coming

Reply:
@fregante I am going to send a lint pr but I only have 2 small things. Have any lint in mind??

Reply:
Can you cleanup some feature names?

- `cleanup` -> `clean`
- drop `add-`

Also if you want to:
- Ensure descriptions always end with `.` (in source or readme)
- Double-check if TODOs have been resolved or can easily be resolved
- Check whether features still work, picking some at random or in any order you'd like

Reply:
> * Change all cache durations to be on one line, i.e. `maxAge: {hours: 1}`

Haha I made then on new lines since I though it was easier to read